docs/how-to-translate-files.md

```diff
@@ -122,6 +122,89 @@ Translating our contributing documentation is a similar flow to translating our
 > [!NOTE]
 > Our contributing documentation is powered by `docsify`, and we have special parsing for message boxes like this one. If you see strings that start with `[!NOTE]`, `[!WARNING]`, or `[!TIP]`, these words should NOT be translated.
 
+## Translate the LearnToCode RPG
+
+The LearnToCode RPG runs on Ren'Py, which uses special syntax for translated strings: (See [Ren'Py Doc on Text](https://www.renpy.org/doc/html/text.html))
```

Comment:
```suggestion
The LearnToCode RPG runs on Ren'Py, which uses special syntax for translated strings: (See [Ren'Py Text documentation](https://www.renpy.org/doc/html/text.html))
```
I think this would make the phrasing a bit less awkward, however, it's probably personal preference.